os/arch/arm/src/amebasmart/amebasmart_boot.c

```diff
@@ -359,16 +359,6 @@ void arm_boot(void)
 
```

Comment:
I see this commit "os/arch/arm/src, os/board/rtl8730e: Minor fixes" is not related to PM.
Could you push this commit in a separate PR please?
It will help review PM changes better and this commit can be merged easier separately as well.

Reply:
Yes, will do